products/opensuse/main.pm

```diff
@@ -891,6 +892,7 @@ else {
         load_fixup_network();
         load_fixup_firewall();
         load_system_update_tests();
+        return 1;
```

Comment:
If the `return` comes here, why do we need the next lines until closing the `unless`?